resources/views/admin/settings.blade.php

```diff
@@ -22,7 +22,7 @@
                     @if (!$value['hidden'])
                         <tr>
                             <td>{{ $key }}
-                                @if ($value['description'] != "")
+                                @if (isset($value['description']) && $value['description'] != "")
```

Comment:
would `!empty($value['description'])` not suffice here

Reply:
works!